gravitee-apim-gateway/gravitee-apim-gateway-core/src/main/java/io/gravitee/gateway/reactive/core/context/HttpRequestInternal.java

```diff
@@ -32,6 +32,11 @@ public interface HttpRequestInternal extends HttpRequest, OnMessagesInterceptor<
      */
     HttpRequestInternal contextPath(final String contextPath);
 
+    // Used only for the API V2 Policy Studio > Debug feature
```

Comment:
```suggestion
    // Used only for the API > Debug feature
```
The V4 Proxy api will soon have a debug feature. that uses this code too, I think.

Reply:
Done